.golangci.yml

```diff
@@ -246,3 +250,34 @@ issues:
     - linters:
         - govet
       text: "shadow: declaration of \"err\" shadows declaration"
+    - path: 'group.go'
+      linters:
+        - interfacebloat
+    - path: 'group/edwards25519/scalar.go'
+      linters:
+        - ineffassign
+        - funlen
+    - path: 'group/edwards25519/const.go'
+      linters:
+        - lll
+    - path: 'group/edwards25519/fe.go'
+      linters:
+        - funlen
+    - path: "share/dkg/pedersen"
+      linters:
+        - gocognit
+        - funlen
+        - gocyclo
+        - cyclop
+    - path: "group/edwards25519/scalar.go"
+      linters:
+        - ineffassign
+    - path: "pairing|group"
+      linters:
+        - revive
+        - stylecheck
+      text: "var-naming: don't use an underscore in package name|ST1003: should not use underscores in package names"
+    - path: "pairing/(circl_bls12381|bn254)/."
```

Comment:
This is a fair warning actually:
https://google.github.io/styleguide/go/decisions.html#package-names

Which means we should rename these public packages before releasing V4 (but not in this PR).

Reply:
https://github.com/dedis/kyber/issues/531